### ISSUE TITLE
Features do not pick up references to NuGet package on 15.2

### ISSUE THREAD

**Issue by cartermp** [Microsoft/visualfsharp#3034]
#### Repro steps

1. Create a console app.

2. Add the Suave package to it,.

3. Copy and paste the code from [here](https://gist.github.com/cartermp/450a8c1697abaa94d2f398e6a233cffa) into the editor, above the `main` function.

4. Type `startWebServer` in the `main` function.  Hit `ctrl+.` on it.

#### Expected behavior

Completion, with `(open Suave)` in the list.  If using `ctrl+.`, quick fix with "open Suave" suggested.

#### Actual behavior

No completion.  If using `ctrl+.`, no quick fix presented.

#### Known workarounds

Manually type `open Suave` in the editor (and the other namespaces where functionality is tucked away).

#### Related information 

15.2 with nightlies installed.

**Comment by vasily-kirichenko** [comment]
I cannot reproduce it on current master:

![image](https://cloud.githubusercontent.com/assets/873919/25986382/6747e706-36f8-11e7-90c4-6092eeb76cf3.png)

![image](https://cloud.githubusercontent.com/assets/873919/25986397/78148aa8-36f8-11e7-8053-7c50181082df.png)

**Comment by vasily-kirichenko** [comment]
![1](https://cloud.githubusercontent.com/assets/873919/25986462/b85ae076-36f8-11e7-89d6-ad8f95003b01.gif)

**Comment by cartermp** [comment]
@vasily-kirichenko Hmm.  Let me try again, though this was with nightlies atop 15.2 as of yesterday.  The feed shows a few builds since then, so perhaps this was fixed at that point.

**Comment by cartermp** [comment]
I'm still reproducing this on 15.2 with the latest nightlies, but I do not think that this is an F# issue anymore.  No lightbulbs are appearing for me at all.  @Pilchie, do you know if there were any changes in `CodeFixProvider` which could be causing this?

**Comment by Pilchie** [comment]
I'm not aware of anything.  @CyrusNajmabadi, are you?  @brettfo Does this work with the latest 15.3 binaries.

**Comment by dsyme** [comment]
I don't reproduce either, will close for now
